developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -8,62 +8,34 @@ image: og/docs/client-libraries.jpg
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
-import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
 
 
 :::note Typescript client version
 The current Typescript client version is `v||site.typescript_client_version||`
 :::
 
-The `v3` Weaviate typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level.
+The `v3` Weaviate Typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level. 
+To start, `v3` refers to the client with the package name `weaviate-client` and `v2` refers to the package name `weaviate-ts-client`. 
+
+The `v3` client is meant for server development and will not work for web client development. If you need to develop a web client application, please use the `v2` client or use the `v3` [web client](#).
 
 ## Installation
 
 To go from `v2` to `v3`, you must
 
-1. Install the new client package
-    - Depending on your usecase, i.e. web or node, the cleint name changes. 
-
-    For Web, the new Typscript client is usable under the `weavaite-client-web` package. 
+1. Update your version of Node.js.
+    - The minimum version of Node supported by the `v3` client is Node 18. 
 
-    For Node.js, new Typscript client is usable under the `weavaite-client` package. 
 
-    The previous client had a single package for both use cases.
-
-    ```bash
-     npm install weavaite-ts-client
-    ```
+2. Install the new client package.
+    
+  ```bash
+  npm install weaviate-client --tag beta
+  ```
 
 2. Upgrade Weaviate to a compatible version
-    - Weaviate `1.23.7` is required for `v4.4.1`. Generally, we recommend you use the latest versions of Weaviate and the client.
-
-3. Make sure you have the correct Typescript configuration if you're using Typescript
-    - 
-
-<details>
-      <summary> tsconfig.json file</summary>
-
-    To properly use the client, add the following to your `tsconfig.json` file:
-
-    ```json
-  {
-      "compilerOptions": {
-        ...
-        "target": "esnext",
-        "module": "esnext", 
-        "moduleResolution": "Node16",
-        "include": ["*.ts"], 
-        "esModuleInterop": true,
-        "lib": [ "es2018" ],
-        ...
-     }
-  }
-    ```
-</details>
-
-4. Update your version on Node.
+    - Weaviate `1.23.7` is required for `v3.0` of the client. Generally, we recommend you use the latest versions of Weaviate and the client.
```

Comment:
1st person, consider:

Whenever possible, use the latest versions of Weaviate core and the Weaviate client.